grading/execute.cpp

```diff
@@ -104,6 +104,8 @@ bool system_program(const std::string &program, std::string &full_path_executabl
     { "spim",                    "/usr/bin/spim" },
     { "clang",                   "/usr/bin/clang" },
     { "gdb",                     "/usr/bin/gdb" },
+    { "iverilog",                "/usr/bin/iverilog" },
```

Comment:
My question is more about the whole docker process then this PR.
* Where is the Submitty docker image defined. Does the image contain the Submitty code base?
* I am wondering why you need to add verlog commands here? My initial thought was they would only be accessible in the docker.

Reply:
Thanks for clarifying.